hedera-mirror-rosetta/scripts/validation/run-validation.sh

```diff
@@ -25,7 +25,8 @@ if (./rosetta-cli 2>&1 | grep 'CLI for the Rosetta API' > /dev/null); then
     echo "Rosetta CLI already installed. Skipping installation"
 else
     echo "Installing Rosetta CLI"
-    curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/master/scripts/install.sh | sh -s -- -b .
+    curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/master/scripts/install.sh | \
+      sh -s -- -b . "${ROSETTA_CLI_VERSION}"
```

Comment:
The default should probably also be in the script in case it's ran outside of CI

```suggestion
      sh -s -- -b . "${ROSETTA_CLI_VERSION:-v0.7.1}"
```

Reply:
maybe not, unless we want to stick with `v0.7.1`.

We should always use the latest (the default if no tag is specified) so as another mechanism to keep our implementation up-to-date.

They should be able to fix the issue pretty soon. This is not the first time they broke the artifact name, I reported a similar bug to them on Sept 8 and they fixed it on the same day.